app/views/courses/articles.json.jbuilder

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 
 json.course do
-  json.articles @course.articles_courses.live.includes(article: :wiki).limit(@limit) do |ac|
+  json.articles @course.articles_courses.all.includes(article: :wiki).limit(@limit) do |ac|
```

Comment:
Is `.all` necessary, or can this just be `@course.articles_courses.includes...` ?